dll/win32/ws2_32/src/sockctrl.c

```diff
@@ -122,6 +122,18 @@ INT
 WSAAPI
 listen(IN SOCKET s,
        IN INT backlog)
+{
+	//Make a separate thread call listen_call()
+	return WSACallThread(1, (LPVOID) s, (LPVOID) backlog);
```

Comment:
Casting variables with a space is completely useless (as per the ReactOS Coding Style).

My review of this Pull Request ends here. Albeit my review was quite a trivial one instead of a real review like involving the detection of possible magical constructs, hopefully I was helpful.

Reply:
This is my first contribute to the project, I'm doing the best I can, your feedback is appreciated, thanks !